core/persistence-jpa/src/main/java/org/apache/syncope/core/persistence/jpa/entity/task/JPATaskExec.java

```diff
@@ -42,6 +43,7 @@
      * The referred task.
      */
     @ManyToOne(optional = false)
+    @JoinColumn(name = "task_id", nullable = false)
```

Comment:
Is this `@JoinColumn` annotation really needed? What if you remove it?

Reply:
Yes, this is needed. Without this the tests `org.apache.syncope.core.persistence.jpa.outer.TaskTest` are going to fail with errors like `Column "TASKEXEC.TASK_ID" not found;`.